benchmarks/inference-server/text-generation-inference/hpa-templates/dcgm-podmonitoring.yaml.tftpl

```diff
@@ -26,7 +26,7 @@ spec:
       app: nvidia-dcgm-exporter
   endpoints:
     - port: metrics
-      interval: 30s
+      interval: 15s
```

Comment:
not a blocker for this PR, wondering if we should turn this into a variable as well to make it easy to play with for future experiments.

Reply:
Filed #783.